cli/src/Api.ts

```diff
@@ -280,7 +306,9 @@ export default class Api {
         let groupMembers: GroupMember[] = [];
         for (let [index, worker] of Object.entries(workers.toArray())) {
             const workerId = workerIds[parseInt(index)];
-            groupMembers.push(await this.groupMember(workerId, worker));
+            if (worker.is_active) {
```

Comment:
Actually the only thing `is_active` does in `Worker` class is checking whether the worker is not an empty object:
```
  get is_active(): boolean {
    return !Boolean(this.isEmpty);
  }
```

`LinkageResult`s coming from `workerById` may contain empty worker object if:
- No workers are available and we try to query all workers with `workerById()` (we then get a `LinkageResult` which is `[[0], EMPTY_WORKER]`)
- We query worker by id with `workerById(ID)` and it doesn't exist

In case of `groupMemebrs` method, this check is probably not even necessary, because the second case is not possible and we have a condition earlier that prevents the first case:
```
        if (nextId.eq(0)) {
            return [];
        }
```

I only named this getter `is_active` for consistancy with `contentWorkingGroup`, where workers actually can either be active or not. In case of `storageWorkingGroup` - inactive workers are just removed from the map (so inactive means "unexisting")